core/shared/src/main/scala/spire/math/Rational.scala

```diff
@@ -58,10 +62,57 @@ sealed abstract class Rational extends ScalaNumber with ScalaNumericConversions
   }
    */
 
-  /* TODO: not commutative
-  def lcm(rhs: Rational): Rational = (lhs / (lhs gcd rhs)) * rhs
-  def gcd(rhs: Rational): Rational
-   */
+  def lcm(rhs: Rational): Rational = if (lhs.isZero || rhs.isZero) Rational.zero else (lhs / (lhs gcd rhs)) * rhs
```

Comment:
nit: space between these 2 methods - perhaps splitting lcm over a couple of lines.